vectorizing/tests/conftest.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+from vectorizing import create_app
+
+@pytest.fixture()
+def app():
```

Comment:
is this fixture being used in any place, or it has another function? or can we perhaps have a single one, eg:

```
@pytest.fixture()
def client(app):
    app = create_app()
    app.config.update({ "TESTING": True })
    return app.test_client()
```